researchstudio-sat/webofneeds
# Feat  remove rate hint

As Requested, this PR:

- Removes the Good Match/Bad Match buttons for hints in favor of including the rating within the sendRequest (only if its a **NON** AdHoc-Connect), and a bad-rating for closing suggested connections.
- Removes the feedback-grid from our codebase
- Also removes all unnecessary (always undefined) parts of send-request.js since we only use this in the visitor-view anyways (no connectionUri was ever in the url)

How To Test:
- Create a (debug-)need
- Get Hints
- Close some of them
- Connect with some of them

You should see that there is never a feedback-grid (Good Match/Bad Match - Buttons) before the request textfield/bad match button

The Button Labels can be adjusted as seen fit, probably directly in github (file: `post-messages.js`), just make sure to include the non-breaking spaces in the simple-chat-textfield label, the close/Bad Match Button does not need non-breaking spaces
